learnosity_sdk/request.py

```diff
@@ -219,3 +222,62 @@ def set_service_options(self):
     def hash_list(self, l):
         "Hash a list by concatenating values with an underscore"
         return hashlib.sha256("_".join(l).encode('utf-8')).hexdigest()
+
+
+class DataApi(object):
+
+    def request(self, endpoint, security_packet,
+                secret, request_packet, action):
+        """Make a single request to Data API"""
+        init = Init('data', security_packet, secret, request_packet, action)
+        return requests.post(endpoint, data=init.generate())
+
+    def request_recursive(self, endpoint, security_packet,
+                          secret, request_packet=None,
+                          action=None):
```

Comment:
I implemented the callback parameter, but decided to take it out because it's the "un-pythonic" way of doing what `request_iter` does. There isn't much of a reason to have both other than to copy the PHP SDK.

Reply:
Yeah, I'd be happy to let it go. Let's see what others think.